website_and_docs/content/documentation/webdriver/interactions/print_page.zh-cn.md

```diff
@@ -159,8 +157,8 @@ Using `getBackground()` and `setBackground()` methods, you can get/set whether b
 {{< /tab >}}
 {{< /tabpane >}}
 
-### ShrinkToFit
-Using `getBackground()` and `setBackground()` methods, you can get/set whether the page will shrink-to-fit content on the page --- boolean `true` or `false`.
+### 缩放至合适大小
+通过 `getBackground()` 和 `setBackground()` 方法，可以获取/设置页面是否会根据页面内容缩小，其为布尔值 `true` 或 `false`。
```

Comment:
This part may be also typo in English original text, should I make a commit in this PR to change that part?

```suggestion
通过 `getShrinkToFit()` 和 `setShrinkToFit()` 方法，可以获取/设置页面是否会根据页面内容缩小，其为布尔值 `true` 或 `false`。
```